org.eclipse.sisu.inject/pom.xml

```diff
@@ -456,7 +456,7 @@
 
   <profiles>
     <profile>
-      <id>sonatype-oss-release</id>
+      <id>sisu-release</id>
```

Comment:
Why not the default profile name `release`?

Reply:
Also, the release profile used by project as not "default" anyway, this PR merely does not change in this area, just shortens its name (and aligns it, with other "release profiles" like `apache-release` or `takari-release` etc).

Reply:
I still don't get why we need a project-specific prefix. I like to have the same profile name across all projects, therefore I usually pick `release`. But as it always has been used with some prefix I am fine with keeping it.